Add Slack and Github Repository to Top Page of ReadTheDocs

tsuchiyama-araya opened the pull request:
## 内容

- Slackのコミュニティーの招待リンク追加
- Githubのリポジトリー追加

<img width="912" alt="Screenshot 2025-03-24 at 15 05 17" src="https://github.com/user-attachments/assets/86ea6d8d-d4b5-4dba-9b97-92edaef05b1e" />

itutu-tienday commented on docs/index.rst:
```diff
@@ -37,3 +37,18 @@ Main Features
+
+Feel free to use it as a space for casual conversations, product questions, requests, and feedback.
+
+Explore Our GitHub Repository
```
It's a minor point, but it would be best to place the GitHub Repository link earlier (giving it priority).

tsuchiyama-araya replied:
I change the position for github repository

tsuchiyama-araya commented:
I Added the slack link to Readme. Also updated ReadTheDocs based on feedbacks.

<img width="999" alt="Screenshot 2025-03-25 at 12 23 43" src="https://github.com/user-attachments/assets/c36628ff-2b04-4ce5-b940-cafdbd59efb8" />
<img width="798" alt="Screenshot 2025-03-25 at 12 31 06" src="https://github.com/user-attachments/assets/8973af27-a78a-4bfa-ae31-6d6e483b0863" />
